docs/releases.md

```diff
@@ -194,11 +193,11 @@ Visit the checklist for more information.
 Additionally, take a look at the [release](https://github.com/jenkins-infra/release/issues?q=is%3Aclosed+label%3Alts-candidate+) and [packaging](https://github.com/jenkinsci/packaging/issues?q=is%3Aclosed+label%3Alts-candidate) repository, for additional LTS candidates.
 
 ### Review tests
-1. Review acceptance tests. Take a look at the checks of your backporting PR from step 10, and make sure, that `Tests / ath / Running ATH / ATH` are green:  
-![acceptance test screenshot](images/jenkins-ath.png)  
-The amount of tests may vary.
-2. Review BOM tests. Make sure, that all tests from your BOM PR are green.
-3. Review JCasC tests. Make sure, that all tests from your JCasC PR are green.
+The following tests need to be reviewed and confirmed to be passing:
+1. **Acceptance tests**: Open a pull request modifying the `jenkins.version` of the `lts` profile in the [acceptance test harness](https://github.com/jenkinsci/acceptance-test-harness/blob/ae19d1e9962f0b162cb3ef9033b24e9629d222c4/pom.xml#L778) suite, to the incremental produced by the backporting pull request in `jenkinsci/jenkins`.
+2. **Plugin compatibility**: Open a pull request modifying the `jenkins.version` in the newest [bom profile](https://github.com/jenkinsci/bom/blob/ce598171d115bfc9dc4d15e6f6c2bbb06e6fe7ed/sample-plugin/pom.xml#L889), to the incremental produced by the backporting pull request in `jenkinsci/jenkins`.
```

Comment:
it needs a file `full-test` created as well or labelled as `full-test` if you have permission:
https://github.com/jenkinsci/bom?tab=readme-ov-file#pct

Reply:
Good point, adding the file is the easiest solution, if you lack triage and write perms.